applications/conversations/views/modules/addpeople.php

```diff
@@ -1,7 +1,7 @@
 <?php if (!defined('APPLICATION')) exit(); ?>
 <div class="Box AddPeople">
     <?php
-    echo panelHeading(t('Add People to this Conversation'));
+    echo panelHeading(t('Add People to this Conversation'), ['aria-level' => '2']);
```

Comment:
Would it be possible to apply the aria-level here inside of the panelHeading function?